components/admin_console/team_channel_settings/team/list/team_row.jsx

```diff
@@ -19,6 +19,32 @@ export default class TeamRow extends React.Component {
         const {team, onRowClick} = this.props;
         onRowClick(team.id);
     }
+
+    renderManagementMethodText = () => {
+        const {team} = this.props;
+        if (team.group_constrained) {
+            return (
+                <FormattedMessage
+                    id={'admin.team_settings.team_row.managementMethod.groupSync'}
```

Comment:
This shouldn't be `{'admin.team_settings.team_row.managementMethod.groupSync'}` use instead just `id='admin.team_settings.team_row.managementMethod.groupSync'`

Reply:
Oops! Not sure how I didn't catch this. Thanks! Fixed 😄